cpp/CMakeLists.txt

```diff
@@ -804,6 +804,13 @@ if(HT_DEFAULT_ALLOCATOR)
     set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -DHT_DEFAULT_ALLOCATOR")
 endif(HT_DEFAULT_ALLOCATOR)
 
+option(PER_THREAD_DEFAULT_STREAM "Build with per-thread default stream" OFF)
```

Comment:
I think you'll need this in the tests and benchmarks cmakelists also.

Reply:
And you need to add a changelog before CI will run.

Reply:
Done.